src/main/scala/fs2/Chunk.scala

```diff
@@ -29,6 +29,16 @@ trait Chunk[+A] { self =>
     iterator.map(f).copyToBuffer(buf)
     Chunk.indexedSeq(buf)
   }
+  /**
+   * `[[scanLeft]]` is a helper for process1.scan; it differs
+   * from standard collection behavior by not including
+   * the initial value in the output.
```

Comment:
Also: I don't think that drop is not necessarily efficient for every chunk type, so since I don't need to rely on it...